### ISSUE TITLE
Need a single place to update the branding and theme color

### ISSUE THREAD

**Issue by roopeshsn** [roopeshsn/freshbey#113]
Freshbey is a self-hostable e-commerce application. Minimum coding experience is enough to host this application by anyone who wishes to create an e-commerce app. Everyone has their own branding and theme color for their e-commerce application. Up to now,  there is no single file to update the branding and theme color. 

Come up with a solution to update the branding and theme across the application using a single file.

**Comment by Athi223** [comment]
Hey, I'd like to give this a try, if its available for contribution

**Comment by roopeshsn** [comment]
Hi, @Athi223 Yeah this issue is still open. Could you explain your approach?

**Comment by Athi223** [comment]
Hey @roopeshsn, I was thinking of storing theme colors in `App.css` or `index.css` inside `:root` and then using those everywhere, what do you think?

**Comment by roopeshsn** [comment]
No, I am not expecting that one. We need a single place to do that!

**Comment by amanycodes** [comment]
Hey, I would like to work on this issue. Can you please assign this to me.

**Comment by Vaishali054** [comment]
hey I would like to give it a try if its available for contribution

**Comment by P-1000** [comment]
hey is this issue still open for contribution.

**Comment by roopeshsn** [comment]
> hey is this issue still open for contribution.

Yep, could you brief your approach?

**Comment by P-1000** [comment]
> > hey is this issue still open for contribution.
> 
> Yep, could you brief your approach?

The plan is to make a single JSON file containing all the settings users can adjust. We'll build a user-friendly web interface where users can easily tweak these settings. Once they've made their adjustments, the web interface will help them create a JSON file with these new settings. This JSON file can then be used throughout the application to update things like branding and colors according to the user's choices. It's designed to be straightforward and user-friendly.
